R/utilities.r

```diff
@@ -280,3 +280,29 @@ dispatch_args <- function(f, ...) {
   formals(f) <- formals
   f
 }
+
+start_servr <- function(serverDirectory = ".", port = 4848,
+                      code = "servr::httd(dir='%s', port=%d)",
```

Comment:
`code` is the same here, as in run_servr. 
`port` is too.
please remove these arguments, because the current repetition is un-necessary, and potentially confusing.